doc/getting_started/setup_fpga.md

```diff
@@ -58,19 +62,19 @@ While Bazel is the entry point for kicking off the FPGA synthesis, under the hoo
 During the build process, the boot ROM is baked into the bitstream.
 As mentioned above, we maintain two boot ROM programs, one for testing (_test ROM_), and one for production (_ROM_).
 
-To build an FPGA bitstream with the _test ROM_, use:
-```console
+To build an FPGA bitstream with the _test ROM_ for the CW340 board, use:
+```sh
 cd $REPO_TOP
-bazel build //hw/bitstream/vivado:fpga_cw310_test_rom
+bazel build //hw/bitstream/vivado:fpga_${BOARD}_test_rom
 ```
 
-To build an FPGA bitstream with the _ROM_, use:
-```console
+To build an FPGA bitstream with the _ROM_ for the CW340 board, use:
```

Comment:
Explain why, even though we're using a generic `$BOARD` variable, the ROM is for the CW340?